cmd/curio/proving.go

```diff
@@ -121,6 +126,40 @@ var wdPostTaskCmd = &cli.Command{
 				break
 			}
 			fmt.Print(".")
```

Comment:
Log?

Reply:
This isn't a Println, just a print appending to the line